### DISCUSSION TITLE
Windows 11 support

### DISCUSSION THREAD

**Discussion by RFNS13** [Fs00/Win10BloatRemover#5] [Questions] [answered]
Hey there,
First of all, let me thank you for making this neat little tool, i've been using it for some time with no issues!

....but then W11 came along. Are you planning to update the app to support it? Both the 'Make System Apps Removable' and the Edge uninstaller (the best features, imo) are not working. I can get around with removing most apps via Powershell commands, but Edge is the stubborn sob we all know about... There's the 'wim-tweak' workaround (can be found on winaero) but it has a history of permanently breaking Windows Update.

Cheers!

**Comment by Fs00** [comment]
Hello @RFNS13! You're welcome 😃 

Unfortunately I don't have any plans to update this tool to support Windows 11.
I'm personally not very interested in upgrading to Windows 11 at this moment (which includes doing the research needed to make this tool work on that version).

That said, I've already played a bit with W11 preview builds and it's possible that I will update the tool in the future, but not anytime soon.

Thanks for understanding.

**Comment by Fs00** [comment, marked as answer]
Hello @RFNS13! I have some good news 😃
I've just released [version 4.3](https://github.com/Fs00/Win10BloatRemover/releases/tag/4.3) of the tool which, among other things, allows _Make system apps removable_ to work on Windows 11. Therefore Edge removal should also fully work on Windows 11.
My stance on fully supporting W11 hasn't changed though, and it's very likely that it won't happen in the foreseeable future. For now, I hope that this little fix will make your W11 experience more bearable 😉
